sos/policies/arch.py

```diff
@@ -0,0 +1,53 @@
+import os
+from sos.policies import PackageManager, LinuxPolicy
+from sos.plugins import Plugin
```

Comment:
Need to import either a newly defined tagging class or one of more of the existing ones.